### PR TITLE
AI Guide v0.1

### PR DESCRIPTION
**Pull request by skyfallsin**
@nathan-barrett styling changes for foxfooding release on Wednesday.

### CODE REVIEW

File: css/ai.css [hunk 1]
```diff
@@ -32,8 +32,25 @@
...
+        /* aspect-ratio: 3 / 1; */
+        height: 305px;
+        display: flex;
+        background: url('/img/ai/ai-header-bg.jpg') no-repeat center right;
+        background-size: cover;
+        align-items: center;
+    }
+
+    #header-content {
+        /* @apply max-h-full max-w-full sm:max-h-80 sm:mx-auto */
```
Comment (nathan-barrett) [inline css/ai.css:46]: If these styles aren't being used any long do they need to be kept? For this line and other comments throughout this file

File: templates/ai/_sidebar.html [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+<script type='text/javascript'>
+  var sidebarDrawer = document.getElementById('my-drawer-2');
+
+  document.querySelectorAll("ul.menu li a").forEach(function(item) {
+    item.addEventListener('click', function() {
+      sidebarDrawer.checked = false;
+      return true;
+    });
+  });
+</script>
```
Comment (nathan-barrett) [inline templates/ai/_sidebar.html:23]: Prefer that this be moved to its own file unless there is a reason it needs to be in the HTML file

**Review by nathan-barrett — APPROVED** [review]
r+wc

**Comment by skyfallsin** [comment]
@nathan-barrett I don't have write access, please deploy

File: templates/ai/content/llms-101/_params-carousel.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  { "alt": "Max Length Parameter Visualization",title": "Max Length", "description": "Setting a limit to the number of tokens.", "image": "/img/ai/carousels/llm-02-maxlength.png"},
+  { "alt": "Temperature Parameter Visualization",title": "Temperature", "description": "Degree of randomness. Setting a high temperature may lead to more creative responses. Setting a low temperature will result in responses that are more literal.", "image": "/img/ai/carousels/llm-03-temp.png"},
+  { "alt": "Top P Parameter Visualization",title": "Top P (Probability)", "description": "Include only the top tokens whose percentage likelihood adds up to the specified “Top P” (e.g. 15%).", "image": "/img/ai/carousels/llm-04-top-p.png"},
+  { "alt": "Frequency Parameter Visualization",title": "Frequency", "description": "The higher this is set, the more that repetition of tokens present in the context will be penalized in suggestions.", "image": "/img/ai/carousels/llm-05-frequency.png"}
+]
+%}
```
Comment (pdehaan) [inline templates/ai/content/llms-101/_params-carousel.html:8]:
```suggestion
{% set slides = [
  { "alt": "LLM Model Parameter Visualization", "title": "Model", "description": "Different models are trained on different sets of data, which influences their response.", "image": "/img/ai/carousels/llm-01-model.png"},
  { "alt": "Max Length Parameter Visualization", "title": "Max Length", "description": "Setting a limit to the number of tokens.", "image": "/img/ai/carousels/llm-02-maxlength.png"},
  { "alt": "Temperature Parameter Visualization", "title": "Temperature", "description": "Degree of randomness. Setting a high temperature may lead to more creative responses. Setting a low temperature will result in responses that are more literal.", "image": "/img/ai/carousels/llm-03-temp.png"},
  { "alt": "Top P Parameter Visualization", "title": "Top P (Probability)", "description": "Include only the top tokens whose percentage likelihood adds up to the specified “Top P” (e.g. 15%).", "image": "/img/ai/carousels/llm-04-top-p.png"},
  { "alt": "Frequency Parameter Visualization", "title": "Frequency", "description": "The higher this is set, the more that repetition of tokens present in the context will be penalized in suggestions.", "image": "/img/ai/carousels/llm-05-frequency.png"}
]
%}
```

**Pull request merged**